src/activity/components/histogram/bars.jsx

```diff
@@ -13,19 +13,19 @@ const dataToWidth = (data, margin, i) =>
 const dataToLeft = (data, margin, i) =>
   `calc((${dataToWidth(data, margin, i)} * ${i}) + ${i * margin}px)`
 
-const valueToPerc = (value, max) => `${value / max * 100}`
-const valueToOpacity = (value, max) => `${valueToPerc(value, max) / 100}`
-const valueToHeight = (value, max) => `${valueToPerc(value, max)}%`
+// const valueToPerc = (value, max) => `${value / max * 100}`
```

Comment:
A lot of commented stuff here. Is there any reason you want to keep this? Maybe this should be documented somewhere?